File: 一些三方库练习/Storyboard和Mansonry测试/XHJ-autoLayout练习/XHJ-autoLayout练习/Classes/ViewController.m
//
//  ViewController.m
//  XHJ-autoLayout练习
//
//  Created by coco on 16/3/25.
//  Copyright © 2016年 XHJ. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    NSLog(@"%@", segue.identifier);
}
- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: DictionaryToModel/DictionaryToModel/ViewController.swift
//
//  ViewController.swift
//  DictionaryToModel
//
//  Created by coco on 16/8/18.
//  Copyright © 2016年 XHJ. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        print(self)
        print(Model.hj_objcetWithDictionary(["1":"2"]))
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

